.travis.yml

```diff
@@ -1,43 +1,57 @@
-language: cpp
-cache: ccache
 os: linux
 dist: xenial
-stages:
-  - check
-  - test
-matrix:
+jobs:
   include:
-    - name: "First commit message adheres to guidelines at <a href=\"https://goo.gl/p2fr5Q\">https://goo.gl/p2fr5Q</a>"
-      stage: check
+    - stage: "Lint and Compile V8" 
```

Comment:
```suggestion
    - stage: "Lint and Compile" 
```

Reply:
Yes, thanks. It's left over from the 3 stage phase of this PR...